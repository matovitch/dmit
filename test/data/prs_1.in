func f()
{
    +a;
    -a;
    a + 3;
    2 * 3.14;
    (((2)));
    -5.2;
    (a + b) * (a - b);
    i >= 3;
    f(-);
    f(3);
    f(a, b);
    f(g());
}
