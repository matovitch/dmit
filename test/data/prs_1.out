{"tree" :{"nodes":[{"kind":"END_OF_TREE","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"FUN_ARGUMENTS","size":0},{"kind":"FUN_RETURN","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_OPPOSE","size":1},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_OPERATOR","size":0},{"kind":"LIT_INTEGER","size":0},{"kind":"EXP_BINOP","size":3},{"kind":"LIT_INTEGER","size":0},{"kind":"EXP_OPERATOR","size":0},{"kind":"LIT_DECIMAL","size":0},{"kind":"EXP_BINOP","size":3},{"kind":"LIT_INTEGER","size":0},{"kind":"LIT_DECIMAL","size":0},{"kind":"EXP_OPPOSE","size":1},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_OPERATOR","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_BINOP","size":3},{"kind":"EXP_OPERATOR","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_OPERATOR","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_BINOP","size":3},{"kind":"EXP_BINOP","size":9},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"EXP_OPERATOR","size":0},{"kind":"LIT_INTEGER","size":0},{"kind":"EXP_BINOP","size":3},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"FUN_CALL","size":1},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"LIT_INTEGER","size":0},{"kind":"FUN_CALL","size":2},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"FUN_CALL","size":3},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"LIT_IDENTIFIER","size":0},{"kind":"FUN_CALL","size":1},{"kind":"FUN_CALL","size":3},{"kind":"SCOPE","size":41},{"kind":"FUN_DEFINITION","size":45},{"kind":"PROGRAM","size":46}],"ranges":[{"start":97,"stop":96},{"start":96,"stop":93},{"start":93,"stop":93},{"start":90,"stop":89},{"start":86,"stop":85},{"start":87,"stop":85},{"start":83,"stop":81},{"start":81,"stop":79},{"start":79,"stop":78},{"start":83,"stop":78},{"start":76,"stop":74},{"start":74,"stop":72},{"start":72,"stop":71},{"start":76,"stop":71},{"start":66,"stop":65},{"start":59,"stop":58},{"start":60,"stop":58},{"start":55,"stop":53},{"start":53,"stop":51},{"start":51,"stop":50},{"start":55,"stop":50},{"start":48,"stop":46},{"start":45,"stop":43},{"start":43,"stop":41},{"start":41,"stop":40},{"start":45,"stop":40},{"start":56,"stop":39},{"start":37,"stop":35},{"start":35,"stop":33},{"start":33,"stop":32},{"start":37,"stop":32},{"start":30,"stop":29},{"start":30,"stop":27},{"start":25,"stop":24},{"start":23,"stop":22},{"start":25,"stop":21},{"start":19,"stop":18},{"start":17,"stop":16},{"start":14,"stop":13},{"start":19,"stop":12},{"start":10,"stop":9},{"start":8,"stop":7},{"start":8,"stop":5},{"start":10,"stop":4},{"start":93,"stop":0},{"start":99,"stop":0},{"start":99,"stop":0}]},"errorSet":{"errors":[]}}